src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/Server/SqlSer.cs

```diff
@@ -146,7 +146,7 @@ internal static SqlUserDefinedTypeAttribute GetUdtAttribute(Type t)
             }
             else
             {
-                throw InvalidUdtException.Create(t, Strings.SqlUdtReason_NoUdtAttribute);
```

Comment:
Passing `Strings.SqlUdtReason_NoUdtAttribute` causes finding `null` which is not desired.
Same for `SqlUdtInfo`.

> _Actual message:_     "'{t.FullName}' is an invalid user defined type, reason: ."
> _Expected message:_ "'{t.FullName}' is an invalid user defined type, reason: **no UDT attribute**."